src/utils/userInfo.ts

```diff
@@ -0,0 +1,17 @@
+import { USER_INFO } from '../constants/localStorage';
+import { IUserInfo } from '../types/auth';
+
+export const setUserInfo = (json: IUserInfo) => {
+  const userInfoString = JSON.stringify(json);
```

Comment:
```suggestion
export const setUserInfo = (userInfo: IUserInfo) => {
  const userInfoString = JSON.stringify(userInfo);
```
> 일관성을 뭔가 계속 강조하게 되는데, 그냥 제 스타일이라 그런 걸 수도 있어요. 
아래 함수에서 userInfo를 쓰고 있어서 동일하게 바꾸면 좋을 것 같아요.

Reply:
json형식으로 param을 넣는다는 의미로 쓴건데 의미해독이 어렵다면 바꾸겠습니다~!